{ 'fall_backward': { 'head': [],
                     'left_arm': [ { 'duration': 0.024,
                                     'name': u'fall_backward',
                                     'positions': [ -2.5101,
                                                    0.8793,
                                                    -0.3882,
                                                    -0.6198,
                                                    -0.1111,
                                                    1.4909,
                                                    0.0233],
                                     'starttime': 0.0}],
                     'left_leg': [ { 'duration': 0.028,
                                     'name': u'fall_forward',
                                     'positions': [ -0.01,
                                                    -0.0943,
                                                    -1.5698,
                                                    2.49,
                                                    0.9111,
                                                    -0.0785],
                                     'starttime': 0.0}],
                     'right_arm': [ { 'duration': 0.026,
                                      'name': u'fall_backward',
                                      'positions': [ 2.5101,
                                                     -0.8793,
                                                     0.3882,
                                                     0.6198,
                                                     0.1111,
                                                     -1.4909,
                                                     -0.0233],
                                      'starttime': 0.0}],
                     'right_leg': [ { 'duration': 0.03,
                                      'name': u'fall_forward',
                                      'positions': [ 0.01,
                                                     0.0943,
                                                     1.5698,
                                                     -2.49,
                                                     -0.9111,
                                                     0.0785],
                                      'starttime': 0.0}],
                     'torso': []}}
